Change README.md to fit new repo objectives

davidomarf opened the pull request:
As discussed via private chat over the last days, `ref` should
serve the purpose of containing a book list that Daque members
find interesting to
a) develop viare, or
b) become better developers

The changes in this commit reflect this new objective by modifying
the proccess of note taking. Books are now categorized as general or
viare-specific.

Commit:
Change README.md to fit new repo objectives

As discussed via private chat over the last days, `ref` should
serve the purpose of containing a book list that Daque members
find interesting to
a) develop viare, or
b) become better developers

The changes in this commit reflect this new objective by modifying
the proccess of note taking. Books are now categorized as general or
viare-specific.

davidomarf commented:
Chuck Norris approves. 👍

quevangel commented on README.md:
```diff
@@ -3,31 +3,84 @@
 This is the place in where we share the list of references we use to keep developing
 [ViaRE](https://github.com/daque-dev/viare).
```
Ya no estará solamente enfocado en Viare.

davidomarf replied:
Preferí dejarlo porque, libro que caiga sobre categoría 1 (ViaRE) o 2 (conocimiento general como desarrolladores), ayuda directa o indirectamente a crear ViaRE.

Si es apropiado, lo quito.

quevangel commented on README.md:
```diff
@@ -3,31 +3,84 @@
 [ViaRE](https://github.com/daque-dev/viare).
 
+This includes both:
+1. books that will be directly used as reference for new integrations into ViaRE, and
```
Lo mismo.

davidomarf replied:
> Incluye. 

La segunda categoría es para libros en general.

Creo que es importante mantner una distinción entre los libros. Por eso propongo la existencia de `/viare/*` y `/*`.

quevangel replied:
Pero bien podría ayudar a algún otro proyecto. No siento que la distinción sea **viare** y **todo lo demás**, preferiría una estructura general con un directorio por proyecto.

davidomarf replied:
Pero ViaRE es, por ahora, el proyecto que más conocimientos fuera-de-la-computación necesita. Cualquier otro proyecto, como `libdaque`, puede recaer en las notas generales.

quevangel reviewed (commented):
Hace falta especificar cómo se distinguirá entre notas de diferentes personas sobre el mismo libro.

davidomarf commented:
> Hace falta especificar cómo se distinguirá entre notas de diferentes personas sobre el mismo libro.

Estarán bajo un subtítulo con el nombre/usuario de quien toma las notas.

```latex
\begin{document}

\section{quevangel}
{notes}

\section{davidomarf}
{notes}
```

Supongo que se debería ser más explícito dentro del README sobre si debe usarse un `section` o un `subsection`.

quevangel commented:
> > Hace falta especificar cómo se distinguirá entre notas de diferentes personas sobre el mismo libro.
> 
> Estarán bajo un subtítulo con el nombre/usuario de quien toma las notas.
> 
> ```
> \begin{document}
> 
> \section{quevangel}
> {notes}
> 
> \section{davidomarf}
> {notes}
> ```
> 
> Supongo que se debería ser más explícito dentro del README sobre si debe usarse un `section` o un `subsection`.

Sí debe ser más explícito.  Y estoy de acuerdo con esa estructura.

Commit:
Reduce ambiguity on adding notes under your name

Specify `\section{{username}}` as the subtitle to add notes under.
Also fix some linebreaks to keep all in less than 90 cols.
